src/main/java/deanery/Visit.java

```diff
@@ -0,0 +1,30 @@
+package deanery;
+
+import deanery.exception.WrongHour;
+import deanery.exception.WrongNumber;
+import deanery.room.Room;
+import deanery.student.Student;
+
+import java.util.List;
+
+
+public class Visit {
+    static List<Room> rooms = RoomData.createRooms();
+    public static void visitation(Student student, String visitRoom, Integer visitHour)
+        throws WrongNumber, WrongHour {
+        for(int i =0; i< rooms.size(); i++){
+            if(visitRoom.equals(rooms.get(i).getNumber()) && student.getType().equals(rooms.get(i).getType())){
+
+                if(visitHour >= rooms.get(i).getOpen() && visitHour<=rooms.get(i).getClose()){
+                    return;

```

Comment:
Writing an empty return in void method is not the best practice -> it is better to handle all exceptions first and if the program does not match to any of the ifs, just don't write any returns